package.json

```diff
@@ -8,6 +8,8 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
+    "lint-fix": "eslint . --ext=js,ts --fix",
```

Comment:
[Note] You can use this command to automatically fix easy-to-fix linting issues 👍 I think we should make `lint-test` part of our CI